### PR TITLE
PJ_SIM_SUNBNB_2023-6 [FE] Registration for users

### PR DESCRIPTION
**Pull request by LennardUy104**
# Issue Link
[PJ_SIM_SUNBNB_2023-6 [FE] Registration for users](https://framgiaph.backlog.com/view/PJ_SIM_SUNBNB_2023-6)

# Definition of done
* Create a markup for the registration page for clients with the components:
    * [x] Username Input Field
    * [x] Email Input Field
    * [x] Password Input Field
    * [x] Sign Up Button
    * [x] Radio Button for Client and Host 
    * [x] if host show upload input for legal documents and ID

# Note
installed vee-validate `npm install vee-validate --save`

to check enter `/register` to url with its domain and port

# Screenshots/Recordings

image
image

### CODE REVIEW

File: fe/src/view/RegistrationView.vue
Comment (andreenriquez11) [inline fe/src/view/RegistrationView.vue:1]: Pages should be in the existing pages folder. No need for a new view folder.

File: fe/src/App.vue
Comment (andreenriquez11) [inline fe/src/App.vue:1]: I think this should be imported in the router page instead? and then just use `<router-view/>` in line 4. `<router-view>` is a special component in Vue.js that is used with Vue Router to render the matched components based on the current URL's route.

**Review by andreenriquez11 — CHANGES_REQUESTED** [review]
- Pages should be in the existing pages folder. 
- Registration Page should be imported in router instead.

File: fe/src/view/RegistrationView.vue
Comment (LennardUy104) [reply fe/src/view/RegistrationView.vue:1]: already moved the registrationView to pages folder

**Review by andreenriquez11 — APPROVED** [review]
lgtm :)

File: fe/src/App.vue [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
-      <HelloWorld />
+      <router-view/>
+      <!-- <RegistrationView/> -->
     </v-main>
   </v-app>
 </template>
 
 <script>
-import HelloWorld from "./pages/HelloWorld.vue";
+  // import RegistrationView from './pages/RegistrationView.vue';
```
Comment (roseaugusto) [inline fe/src/App.vue]: @LennardUy104 Please avoid unnecessary comments, please confirm if this is still needed. If not, please remove.

**Review by roseaugusto — CHANGES_REQUESTED** [review]

File: fe/src/App.vue (hunk 1 above)
Comment (LennardUy104) [reply fe/src/App.vue]: @roseaugusto Already resolve changes

**Review by roseaugusto — APPROVED** [review]

**Pull request merged**